scripts/update-dependencies/Program.cs

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.DotNet.VersionTools;
+using Microsoft.DotNet.VersionTools.Automation;
+using Microsoft.DotNet.VersionTools.Dependencies;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace Microsoft.Dotnet.Scripts
+{
+    public static class Program
+    {
+        private static readonly Config config = Config.Instance;
+
+        public static void Main(string[] args)
+        {
+            Trace.Listeners.Add(new TextWriterTraceListener(Console.Out));
+
+            if (!ParseArgs(args))
+                return;
+
+            CreatePullRequest();
+        }
+
+        private static bool ParseArgs(string[] args)
+        {
+            foreach (string arg in args)
+            {
+                int idx = arg.IndexOf('=');
+                if (idx < 0)
+                {
+                    Console.Error.WriteLine($"Unrecognized argument '{arg}'");
+                    return false;
+                }
+
+                string name = arg.Substring(0, idx);
+                string value = arg.Substring(idx + 1);
+                Environment.SetEnvironmentVariable(name, value);
+            }
+
+            return true;
+        }
+
+        private static void CreatePullRequest()
+        {
+            GitHubAuth gitHubAuth = new GitHubAuth(config.Password, config.UserName, config.Email);
+            GitHubProject origin = new GitHubProject(config.GitHubProject, config.UserName);
+            GitHubBranch upstreamBranch = new GitHubBranch(config.GitHubUpstreamBranch, new GitHubProject(config.GitHubProject, config.GitHubUpstreamOwner));
+
+            string commitMessage = "Updating external dependencies";
```

Comment:
Can we include more information in the commit message and pull request description?

Reply:
What info would you like? Only thing I can come up with is the versions that were changed. That could go in the description, but not really the PR title or commit message.

Reply:
As mentioned in person, I think having the orchestrated build ID as found here would be sufficient:

https://github.com/dotnet/versions/blob/master/build-info/dotnet/product/cli/release/2.1/build.xml#L1. Example contents:
```xml
<OrchestratedBuild Name="cli" BuildId="20180218-02-1397458">
```